components/sysutils/lsof/Makefile

```diff
@@ -30,6 +31,9 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/justmake.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+# Need GNU version of yes
+PATH = $(PATH.gnu)
+
```

Comment:
Yes, it sounds like the same bug.  As I mentioned, I'm building under oi_151a8, the most recent OI version available on the SPARC hardware platform.  I do need the workaround.  People building under hipster for x86 hardware don't need it, but that workaround shouldn't cause any problems for them.  Shall I make that comment more specific?